Run_VEP_batch.sh

```diff
@@ -105,17 +105,21 @@ if [ $(df -P . | tail -1 | xargs | cut -d" " -f4) -lt 5000000 ];then
   printf "INFO: There is less then 5GB space left on your disk, please check\n"
 fi
 
+#for every file that ends with .vcf that stayed there for longer than timeout (that value is stored in the config)
 #check for whitespaces and repaces them with underscore
 SAVEIFS=$IFS
 IFS=$'\n'
-for f in $(find input/ -name '*\ *.vcf')
-do
-  mv "$f" "${f// /_}"
-  printf "I don't like whitespaces: $f is renamed to ${f// /_}"
-done
+for f in $(find input/ -name '*\ *.vcf' -mmin +$waitperiod)
```

Comment:
Also, is there a reason why renaming files is performed in a separate for-loop?
Otherwise renaming could be performed in the following loop (Line 123) as the current implementation needs the double amount of time as both `find`-commands need to check for the wait period which will be reseted after moving files.